### PR TITLE
Positions updated

### PR DESCRIPTION
**Pull request by Abhinav200207**
### Fixes #<number>
<Do not delete this template>
Description: <provide a description> The title should be short and relevant.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

### CODE REVIEW

**Review by SrijanSriv — CHANGES_REQUESTED** [review]
Thanks for updating the members @Abhinav200207 .

You might want to check out the deployment preview on different screens like a phone. There are some css related aspects that may also be cleared. 
Also, try to maintain a hierarchy. The coordinators go side by side, all below the head; for each community.

**Review by SrijanSriv — APPROVED** [review]
LGTM. Thanks a lot @Abhinav200207 :whale:

File: client/src/components/Members/Community/WebDAppD.members.js [hunk 1]
```diff
@@ -1,20 +1,116 @@
...
+            window.addEventListener('resize', handleResize);
+            return () => window.removeEventListener('resize', handleResize);
+      }, []);
+
+
+      return (windowDimensions.width > 990 ? (<div className="electronics-members">
+            <hr className="header-line-top" />
+
+            <h1 className="community-header">
+                  Core Electronics Community Members
```
Comment (EmperorYP7) [inline client/src/components/Members/Community/WebDAppD.members.js:37]: Change it to `Software Development Wing`

**Review by 0548929024 — COMMENTED** [review]
spam

**Pull request merged**